domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewService.kt

```diff
@@ -55,19 +54,6 @@ class ReviewService(
       prisonNumber,
     )
 
-  /**
-   * Returns a list of [ReviewSchedule] for the prisoner identified by their prison number.
-   */
-  fun getReviewSchedulesForPrisoner(prisonNumber: String): List<ReviewScheduleHistory> {
```

Comment:
I wonder if the method that calls this should also be in it's own controller too - rather than ReviewController?